lib/solaar/ui/window.py

```diff
@@ -538,11 +522,11 @@ def _details_items(device, read_all=False):
 				for fw in list(device.firmware):
 					yield ('  ' + _(str(fw.kind)), (fw.name + ' ' + fw.version).strip())
 			elif device.kind is None or device.online:
-				yield ('  %s' % _("Firmware"), '...')
+				yield (_("  Firmware"), '...')
 
 			flag_bits = device.status.get(_K.NOTIFICATION_FLAGS)
 			if flag_bits is not None:
-				flag_names = ('(%s)' % _("none"),) if flag_bits == 0 else _hidpp10.NOTIFICATION_FLAG.flag_names(flag_bits)
+				flag_names = _("(none)") if flag_bits == 0 else _hidpp10.NOTIFICATION_FLAG.flag_names(flag_bits)
```

Comment:
Same here, the format is normally always `(X)`, so I guess it makes more sense to make `X` a translatable string